docs/java-api/docs/index_.asciidoc

```diff
@@ -140,7 +140,7 @@ String json = "{" +
     "}";
 
 IndexResponse response = client.prepareIndex("twitter", "tweet")
-        .setSource(json)
+        .setSource(json,XContentType.JSON)
```

Comment:
nit: Could you add a whitespace afther the comma for better readability?

Reply:
hi, @cbuescher I'm so sorry about my doc style,this is my first commit for elasticstatic. and shoud I create a new pull request again?